### DISCUSSION TITLE
Create a name that is already taken by a seemingly dormant organisation

### DISCUSSION THREAD

**Discussion by andimon** [community/community#144805] [Other Feature Feedback, Questions, & Ideas]
Hi,

I’m interested in creating a GitHub organization, but the name I want is already taken. The organization appears dormant. Is there a way to contact GitHub or the organization to request the use of that name?

Thanks!

**Comment by shinybrightstar** [comment]
Hi @andimon,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
